src/openmethane_prior/layers/omGFASEmis.py

```diff
@@ -51,6 +52,10 @@ def download_GFAS(
 ):
     """Download GFAS methane between two dates startDate and endDate, returns nothing"""
     dateString = start_date.strftime("%Y-%m-%d") + "/" + end_date.strftime("%Y-%m-%d")
+
+    downloadPath = pathlib.Path(file_name);
+    downloadPath.parent.mkdir(parents=True, exist_ok=True)
```

Comment:
The `cdsapi` client is not smart enough to create the containing path before attempting to download. This fixes an issue when running the test_gfas_emis test or running `omGFASEmis.py` standalone in a path with no `intermediates` folder.

This folder prep might be better handled in the config but I wanted to touch as little as possible.

Reply:
Thats fine. This is a noop if the directory already exists